tests/test_astropy.py

```diff
@@ -45,8 +45,23 @@ def test_angle():
 def test_basic():
     """Basic tests of CelestialCoord construction and corresponding SkyCoord construction.
     """
-    pass
 
+    t0 = time.time()
+    c1 = coord.CelestialCoord(0.234 * coord.radians, 0.342 * coord.radians)
+    t1 = time.time()
+
+    a1 = astropy.coordinates.SkyCoord(0.234, 0.342, unit=units.radian,
+                                      frame=astropy.coordinates.FK5(equinox='J2000'))
```

Comment:
I'm pretty sure the frame isn't actually relevant here.  You could leave this at the default (which I believe is ICRS).